src/main/java/cli/CliProperties.java

```diff
@@ -39,7 +39,17 @@
   private String outputFile = "route.json";
 
   @CommandLine.Option(
-      names = {"-c", "config"},
-      description = "output csv configuration: s -standard , c -custom ")
-  private char config = 's';
+      names = {"-c", "-config"},
+      description = "output csv configuration path")
+  private String configPath = "configuration.json";
+
+  @CommandLine.Option(
+      names = {"-r", "results"},
+      description = "csv result file")
+  private String csvPath = "csvRouteRequest.csv";
```

Comment:
maybe you can remove the csv from the file name? it is already in the file extension?